src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -58,12 +59,36 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             DocumentCollectionInfo documentCollectionLocation;
             DocumentCollectionInfo leaseCollectionLocation;
             ChangeFeedProcessorOptions processorOptions = BuildProcessorOptions(attribute);
+            
             processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
             if (attribute.MaxItemsPerInvocation > 0)
             {
                 processorOptions.MaxItemCount = attribute.MaxItemsPerInvocation;
             }
 
+            var startFromTime = DateTime.UtcNow;
+
+            if (!string.IsNullOrEmpty(attribute.StartFromTime) && 
+                !DateTime.TryParse(attribute.StartFromTime, out startFromTime))
+            {
+                throw new InvalidOperationException("Specified StartFrom parameter is not in appropriate DateTime format. Please refer: https://docs.microsoft.com/dotnet/api/system.datetime.parse?view=net-5.0#parsing-and-cultural-conventions");
+            }
+
+            if (attribute.StartFromBeginning && 
+                !string.IsNullOrEmpty(attribute.StartFromTime))
+            {
+                throw new InvalidOperationException("Only one of StartFromBeginning or StartFromTime can be used");
+            }
+
+            processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
+            if (!attribute.StartFromBeginning &&
+                !string.IsNullOrEmpty(attribute.StartFromTime))
```

Comment:
I think you can simply check for the StartFromTime, at this point, you already validated that StartFromBeginning cannot be true if StartFromTime is not null.